common/minja/minja.hpp

```diff
@@ -240,7 +252,7 @@ class Value : public std::enable_shared_from_this<Value> {
       auto index = key.get<int>();
       return array_->at(index < 0 ? array_->size() + index : index);
     } else if (object_) {
-      if (!key.is_hashable()) throw std::runtime_error("Unhashable type: " + dump());
+      if (!key.is_hashable()) throw std::runtime_error("Unashable type: " + dump());
```

Comment:
```suggestion
      if (!key.is_hashable()) throw std::runtime_error("Unhashable type: " + dump());
```
I guess this was merged here previously, perhaps you'll upstream it?

Reply:
Ah, thanks! Yes I'd rather only do changes in the minja repo and just sync back wholesale here, I'll apply these there shortly.